client/disputes/index.tsx

```diff
@@ -185,7 +185,7 @@ export const DisputesList = (): JSX.Element => {
 				),
 			},
 			reason: {
-				value: dispute.reason,
+				value: reasonDisplay,
```

Comment:
This looks nicer in the CSV and the more decriptive name aligns with other column values, e.g. Status = "Needs Response".

There is a risk that merchants may rely on previous values. We could create an additional column (e.g. "Reason Description").

It is probably a small risk and see your change as an improvement.

Reply:
Valid concern that if someone relies on the value of that column might break their workflow but I think it's OK to 'break' here because we're sticking with the new wording for long term.